### ISSUE TITLE
PR: Revit 2023 Cannot export property set for IfcProject

### ISSUE THREAD

**Issue by Moult** [Autodesk/revit-ifc#634]
### Problem Description

We have a user defined property sets.txt file which defines our own property set like this:

```
PropertySet:    Foo_bar    I    IfcProject
    Foobar Label Project Name
```

(note: tabs are used, just typing in manually here so spaces are used instead)

It's not common to have properties on IfcProject, but this is a contractual requirement for a large client and they are checking :)

With this setup in Revit 2022 using the latest plugin from Github it works fine and I can see the Project Name parameter in the Project Information show up in the Foo_bar.

On Revit 2023, nothing shows up.

No test file is needed as you can recreate this on any new empty project with the built in Project Name parameter.

### Revit Version

2023.1.x

### IFC for Revit Addon Version

23.2.5

### Windows Version

11 22H2

**Comment by AngelVelezSosa** [comment]
Hi @Moult , that is a known regression in 2023 specific to IfcProject that will be fixed in the next update.

**Comment by Moult** [comment]
Awesome, thanks for the super fast response. I'll let them know to leave it as is, and we'll wait for the next update.

**Comment by AngelVelezSosa** [comment]
Closing this as a duplicate of #602.  We are preparing the fix now.

**Comment by dvrvb** [comment]
It is indeed fixed in version 23.3.0.0.
